### PR TITLE
Add cookie password env to app.json

### PR DESCRIPTION
**Pull request by gideonthomas**
aForgot to update this when I added a new env var

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -87,6 +87,9 @@
     "SIGNUP": {
       "required": true
     },
+    "SECRET_COOKIE_PASSWORD": {
+      "required": true
```
Comment (cadecairos) [inline app.json]: lets add `"generator": "secret"` to this, it doesn't need to inherit this secret from the staging environment.

Comment (cadecairos) [reply app.json]: Are there other places we can us the generator, so that we're not re-using staging secrets?

Comment (gideonthomas) [reply app.json]: good catch!

Comment (gideonthomas) [reply app.json]: don't think there are other instances

**Pull request merged**